lib/HALTarget/src/Network.cpp

```diff
@@ -196,6 +218,16 @@ bool Network::connect()
         LOG_DEBUG("MQTT client connected to broker");

```

Comment:
Isnt't that asynchronous?

Reply:
I do not think so: https://pubsubclient.knolleary.net/api#connect

Reply:
The API doesn't describe whether its asynchronous or not. Thats because it depends on the client which is used!